### ISSUE TITLE
Replace Calypso "Posts" with the standard WordPress "Posts" view

### ISSUE THREAD

**Issue by richtabor** [Automattic/wp-calypso#95621]
> [!IMPORTANT]
> **Project thread: pekYwv-4PY-p2**
> Tracked by #98483

As part of the Untangling Calypso epic, let's remove the custom implementation of the wordpress.com/posts view, in lieu of the standard WordPress `wp-admin/edit.php` view—for all users.

This will reduce confusion navigating between Calypso and proper WordPress admin.

Requirements

- Selecting "Posts" from the WordPress or Calypso menu opens the standard WordPress posts view at  `wp-admin/edit.php`

Please see pdtkmj-3dx-p2

**Comment by tanjoymor** [comment]
**Recommendation:** Switch to WP Admin for All users / All platforms

**Considerations:**

Calypso offers an easy “Copy link” option that is missing in WP Admin
* Add to WP Admin? Low priority.
* Happiness should monitor if we see an uptick in users linking content to the wrong URL (aka the editor instead of the live link)

Calypso offers a “QR Code” option
* This feature shows you a QR code, which perhaps makes it easier to open the post on your phone from your computer, but you can’t make a copy of this QR Code and actually “use” it anywhere.
* Could this be excluded? Not sure if we have any data on its usage.

The Calypso “Share” posts link behaves differently than the WP Admin Share link
* This should probably be addressed sooner rather than later. The WP Admin functionality is confusing as it opens the post editor and isn’t immediately apparent that the Jetpack sidebar has been triggered.
* If we want to keep this functionality, instead of the Calypso functionality, we could perhaps add a tooltip that draws attention to the Jetpack sidebar and sharing options.

_Note: The Copy Link and QR Code are also part of Pages, Portfolio, and Testimonials screens_

**Comment by Copons** [comment]
I don't see any blockers for this. Let's use **wp-admin for new users**, as recommended, for both Posts and its sub-screens!

### Considerations

- Copy Link
  - wp-admin has the "View" quick action that performs a similar duty. I don't consider the "copy link" significant enough to think twice about losing it.
  - If HEs observe any upticks, adding "Copy Link" to the quick actions is technically trivial.
- QR Code
  - **The feature exists on wp-admin Atomic already! Let's port it to Simple.**
  - And let's do it as part of the duplication removal.
- Share
  - **The feature exists on wp-admin Atomic already! Let's port it to Simple.**
  - And let's do it as part of the duplication removal.
  - @tanjoymor I agree that the wp-admin version is not as nice as the Calypso one. But that's a different issue. 🙂

**Comment by tanjoymor** [comment]
@Copons fair enough on the Share feature. 

Is it relatively easy to include a tooltip as part of this work?

IMO there's not enough of a visual queue in the current wp-admin state for users to understand what happened when they clicked "Share". Especially important for existing users, but more helpful and intuitive for new users too.

**Comment by mmtr** [comment]
> for new users

We discussed this today and decided that, considering we'll release changes with a holdout test, we want to target all users (new and existing), so I updated the issue summary to reflect that. See p1733745514693799-slack-C04DZ8M0GHW.

**Comment by ederrengifo** [comment]
Regarding the share functionality, I recently posted a few ideas that could users to not lose this functionality. Maybe it's worth to allocate some efforts to address it from Jetpack: https://jetpackp2.wordpress.com/2024/12/14/untangling-calypso-should-we-work-on-reshare-functionality/ cc @oskosk

**Comment by richtabor** [comment]
> The Calypso “Share” posts link behaves differently than the WP Admin Share link

@ederrengifo Is this the share functionality we're referring to? 

https://github.com/user-attachments/assets/2158619a-dae9-4da8-a59e-4f4874dea55c

**Comment by richtabor** [comment]
Is above the "core" version of this? 

![Image](https://github.com/user-attachments/assets/2c99ff7c-ff76-4984-ad6a-affb50e2cc4b)

**Comment by mmtr** [comment]
@getdave this is already handled by https://github.com/Automattic/jetpack/pull/40479 and https://github.com/Automattic/jetpack/pull/40568.

The issue is open because I think @BogdanUngureanu was porting a missing Calypso feature to wp-admin, but this is pretty much done, so you can pick a different issue.

**Comment by getdave** [comment]
I've just updated this Issue description to include the link to pdtkmj-3dx-p2 as that was missing which is why I started down the rabbit hole.

**Comment by pablinos** [comment]
Yes, this is the latest discussion about the resharing functionality p1HpG7-vji-p2. It's been an ongoing debate for a number of years.

Pushing people into the editor for resharing is ok, for one use case, but not ideal, and we still need to solve the interface for scheduled resharing. In the longer term I'd prefer us to swap to using a CPT for social shares, and develop a more general purpose social media tool, with a calendar/timeline interface, but that obviously isn't for now!

I don't know how easy it would be to rebuild the resharing UI - or some variation of it - to be shown in a modal on the Post list screen. My guess is it's not worth the effort, given the low usage, but I also think the feature is buried rather than it not adding value. There were also some concerns about performance, and conflicts with plugins, when we first looked at this, but I think we probably have patterns for loading React UI in the admin pages now.
